tribuo-tutorials/README.md

```diff
@@ -0,0 +1,35 @@
+# Tribuo tutorials
+
+This project shows how to run [Tribuo Machine Learning](https://tribuo.org) algorithms with Conclave.
+
+The code is based on [Tribuo's tutorials](https://tribuo.org/learn/4.0/tutorials/), which cover the following examples:
+* [Classification](https://tribuo.org/learn/4.0/tutorials/irises-tribuo-v4.html)
+* [Clustering](https://tribuo.org/learn/4.0/tutorials/clustering-tribuo-v4.html)
+* [Regression](https://tribuo.org/learn/4.0/tutorials/regression-tribuo-v4.html)
+* [Anomaly Detection](https://tribuo.org/learn/4.0/tutorials/anomaly-tribuo-v4.html)
+* [Configuration](https://tribuo.org/learn/4.0/tutorials/configuration-tribuo-v4.html)
+
+In this sample the client is using Mail to communicate with the enclave.
```

Comment:
Which sample? Is this a tutorial or a sample?

Reply:
I've removed sample and stuck with tutorials. Let me know if I missed it anywhere.